lib/apollo/withData.js

```diff
@@ -80,6 +81,12 @@ export default ComposedComponent => {
       }
     }
 
+    componentDidMount () {
```

Comment:
@patte @tpreusse @garamond 

I would like to discuss these lines.
The main purpose of this is having access to the `Router` instance in the app, so I can push routes client-side, without reloading the page as it is happening right now when you click on the native header ([here](https://github.com/orbiting/app/pull/100/files#diff-5a62d3a9fde9e73420c622a15da32f91R61))

However,  I _think_ this can be considered a security flaw, because this would be accessible from the console. It's not that someone can do anything I think (accessing a route that he does not have access). We can trigger this line only when app running in app also to make it more private.

But anyways, I would like to hear your thoughts. What I can say is that the app works MUCH nicer with this client-side redirection than without it.

Reply:
I leave that to the FE specialists to decide.